src/actions/getConjunctionEventAdditionalRisk.ts

```diff
@@ -0,0 +1,24 @@
+'use server';
+
+export type AdditionalObjectRisk = {
+  objectName: string;
+  currentWeeklyEvents: number;
+  totalEventsFollowingWeek: number;
+  additionalEventsFollowingWeek: number;
+};
+
+type ReportResponseData = {
+  additional_object_risk: AdditionalObjectRisk[];
+};
+
+export async function getConjunctionEventAdditionalRisk(presignedUrl: string) {
+  const data = await fetch(presignedUrl);
+  const reports: ReportResponseData[] = await data.json();
+  const lastReport = reports[0];
```

Comment:
Could it be undefined? If yes we need to add

```suggestion
  const lastReport = reports.length && reports[0];
```

Reply:
if we have 200 return this always will be array even if empty so if i take [0] i will get undefined